src/base/bittorrent/session.h

```diff
@@ -761,6 +761,10 @@ namespace BitTorrent
         QStringMap m_categories;
         QSet<QString> m_tags;
 
+        // I/O errored torrents
+        QVector<InfoHash> m_recentErroredTorrents;
+        QTimer *m_recentErroredTorrentsTimer;
```

Comment:
Aren't there too many timers? I would use the existing one. I mean torrents update timer.

Reply:
Or can you change it to be just `QTimer`? no point for pointer type.

Reply:
>Aren't there too many timers? I would use the existing one. I mean torrents update timer.

I could do that if it is ok to suppress errors that long... (can't find the timeout duration now)

>Or can you change it to be just QTimer? no point for pointer type.

There are other member in the same header that are QTimer pointers.
I don't wish to increase compilation time or expose the QTimer header to others who include the session.h heaer.